cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -219,6 +229,11 @@ func (s *VMTServer) Run() {
 	// the certificate issue of 'doesn't contain any IP SANs'.
 	// See https://github.com/kubernetes/kubernetes/issues/59372
 	kubeletClient := s.createKubeletClientOrDie(kubeConfig, !isOpenshift)
```

Comment:
Based on the fact that you said it worked for you when you changed `!isOpenShift` to `isOpenShift`, it means your server has valid CA certs and there shouldn't be a need to ForceSelfSignedCerts?